test/ClaimTest.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types=1);
+
+namespace LaminasTest\Di;
+
+use LaminasTest\Di\TestAsset\ClaimTestClass;
+use PHPUnit\Framework\TestCase;
+use ReflectionClass;
+
+use function array_values;
+use function uasort;
+
+final class ClaimTest extends TestCase
```

Comment:
This kinda work, but it should be in `ClassDefinitionTest` (if there is such test) 🤔